api/server/server.go

```diff
@@ -1482,27 +1550,31 @@ func createRouter(s *Server) *mux.Router {
 		ProfilerSetup(r, "/debug/")
 	}
 	m := map[string]map[string]HttpApiFunc{
+		"HEAD": {
+			"/containers/{name:.*}/archive-path": s.getContainersArchivePath,
```

Comment:
can we have a proper head resource? Why do we need to create the tar file when it's going to be discarded?